spec/controllers/managements_controller_spec.rb

```diff
@@ -43,12 +68,34 @@
       end
     end
 
+    describe 'POST /managements/site_update' do
+      it 'works!' do
+        data = [{"var"=>"site.Neta1", "value"=>"Toro"}, {"var"=>"site.Neta2", "value"=>"Unit"}]
```

Comment:
__[rubocop]__
* Style/SpaceInsideHashLiteralBraces: Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceAroundOperators: Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceAroundOperators: Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceInsideHashLiteralBraces: Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/SpaceInsideHashLiteralBraces: Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceAroundOperators: Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceAroundOperators: Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/SpaceInsideHashLiteralBraces: Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)